src/reducers/_send.js

```diff
@@ -57,18 +59,17 @@ const SEND_CLEAR_FIELDS = 'send/SEND_CLEAR_FIELDS';
 export const sendModalInit = () => (dispatch, getState) => {
   const { accountAddress, accountInfo, prices } = getState().account;
   const { gasLimit } = getState().send;
-  const selected = accountInfo.assets.filter(
-    asset => asset.symbol === 'ETH',
-  )[0];
   const fallbackGasPrices = parseGasPrices(null, prices, gasLimit);
+
   dispatch({
     type: SEND_GET_GAS_PRICES_REQUEST,
     payload: {
       address: accountAddress,
-      selected,
       gasPrices: fallbackGasPrices,
+      selected: {},
```

Comment:
@jinchung - I could be wrong, but for the intentions of the mobile send screen I do not believe that we want this to be defaulted to `ETH`.

Reply:
cool - let's just default it to nothing and have an option to specify a value so that Manager can have a defaulted value